src/thorlabs_cube/driver/kcube/kpz.py

```diff
@@ -0,0 +1,193 @@
+        configuration parameters for TRIG1 and TRIG2 (mode, polarity).
+        Purpose description can be found in set_trigio_config()
+        """
+        get_msg = await self.send_request(
+            MGMSG.KPZ_REQ_KCUBETRIGIOCONFIG, [MGMSG.KPZ_GET_KCUBETRIGIOCONFIG], _CHANNEL
+        )
+        return st.unpack("<HHHHHHHHHHH", get_msg.data[6:])
+
+
+class KpzSim(TpzSim):
+
+    def set_kcubemmi_params(
+        self,
+        js_mode: int,
+        js_volt_gearbox: int,
+        js_volt_step: int,
+        dir_sense: int,
+        preset_volt1: int,
+        preset_volt2: int,
+        disp_brightness: int,
+        disp_timeout: int,
+        disp_dim_level: int,
+    ) -> None:
+        """Set the KCube MMI parameters in simulation.
+
+        This simulates setting the parameters for the joystick, voltages, and display.
+        """
+        self.js_mode = js_mode
+        self.js_volt_gearbox = js_volt_gearbox
+        self.js_volt_step = js_volt_step
+        self.dir_sense = dir_sense
+        self.preset_volt1 = preset_volt1
+        self.preset_volt2 = preset_volt2
+        self.disp_brightness = disp_brightness
+        self.disp_timeout = disp_timeout
+        self.disp_dim_level = disp_dim_level
+
+    def get_kcubemmi_params(
+        self,
+    ) -> Tuple[int, int, int, int, int, int, int, int, int, int, int, int, int]:
+        """Get the KCube MMI parameters in simulation.
+
+        This returns the simulated parameters for the joystick, voltages, and display.
+        """
+        return (
+            self.js_mode,
+            self.js_volt_gearbox,
+            self.js_volt_step,
+            self.dir_sense,
+            self.preset_volt1,
+            self.preset_volt2,
+            self.disp_brightness,
+            self.disp_timeout,
+            self.disp_dim_level,
+            _RESERVED,
+            _RESERVED,
+            _RESERVED,
+            _RESERVED,
+        )
+
+    def set_trigio_config(
+        self, trig1_mode: int, trig1_polarity: int, trig2_mode: int, trig2_polarity: int
+    ) -> None:
+        """Set the TRIG1 and TRIG2 configuration in simulation.
+
+        This simulates setting the TRIG1 and TRIG2 modes and polarities.
+        """
+        self.trig1_mode = trig1_mode
+        self.trig1_polarity = trig1_polarity
+        self.trig2_mode = trig2_mode
+        self.trig2_polarity = trig2_polarity
+
+    def get_trigio_config(
+        self,
+    ) -> Tuple[int, int, int, int, int, int, int, int, int, int]:
+        """Get the TRIG1 and TRIG2 configuration in simulation.
+
+        This returns the simulated configuration for the TRIG1 and TRIG2 modes and polarities.
+        """
+        return (
+            self.trig1_mode,
+            self.trig1_polarity,
+            self.trig2_mode,
+            self.trig2_polarity,
+            _RESERVED,
+            _RESERVED,
+            _RESERVED,
+            _RESERVED,
+            _RESERVED,
+            _RESERVED,
+        )
```

Comment:
```suggestion
        )

```
Newline required at the end of this file according to [most recent failed linting workflow.](https://github.com/quantumion/thorlabs_cube/actions/runs/11544520821/job/32130182167)